scripts/check.py

```diff
@@ -85,6 +85,7 @@ def fix(self, commit):
 format_file_types = OrderedDict(
     {
         "CMakeLists.txt": attrdict({"formatter": CMakeFormatter}),
+        "ThirdpartyToolchain.cmake": attrdict({"formatter": CMakeFormatter}),
```

Comment:
Can we just make it *.cmake to make it more future proof.

Reply:
Thanks, I have done!